Resume.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Emma's Resume</title>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+
+  <header>
+    <h1 class="header--headline">My Qualifications</h1>
+  </header>
+
+<hr class="main-body--hr__top">
+
+<div class="main-body">
+
+  <article>
+    <img class="main-body__portrait" src="images/emma-resume.png">
```

Comment:
Although not required, the best practice is to include the `alt` attribute which allows for accessibility to the visually impaired and is what appears if the resource fails to load.  🔨